src/Core/src/Platform/iOS/LabelExtensions.cs

```diff
@@ -73,7 +73,7 @@ public static void UpdateLineHeight(this UILabel platformLabel, ILabel label)
 				platformLabel.AttributedText = modAttrText;
 		}
 
-		internal static void UpdateTextHtml(this UILabel platformLabel, ILabel label)
+		internal static void UpdateTextHtml(this UILabel platformLabel, ILabel label, double fontSize)
```

Comment:
It is not necessary to pass this parameter since `ILabel ` is already passed, which has access to the Font property.

Reply:
Thanks, I've fixed that :)